src/libraries/ChainlinkDataFeedAltLib.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import {AggregatorV3Interface} from "../interfaces/AggregatorV3Interface.sol";
+
+import {ErrorsLib} from "./ErrorsLib.sol";
+
+struct Quantity {
```

Comment:
maybe we should specify what `decimals` are we talking about, the feed? The token?